proto/src/main/proto/pb.proto

```diff
@@ -1321,6 +1321,8 @@ message PersistableEnvelope {
 
         XmrAddressEntryList xmr_address_entry_list = 1001;
         SignedOfferList signed_offer_list = 1002;
+
+        EncryptedMoneroConnectionStore xmr_connection_store = 2001;
```

Comment:
Remove newline, rename to `EncryptedXmrConnectionList` for consistency with `XmrConnectionList`, change assigned number to `1003`.

`XmrConnectionList` doesn't need to be listed too?